pkgs/development/python-modules/natural/default.nix

```
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+}:
+buildPythonPackage rec {
+  pname = "natural";
+  version = "0.2.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "tehmaze";
+    repo = "natural";
+    rev = "${version}";
+    hash = "sha256-DERFKDGVUPcjYAxiTYWgWkPp+Myd/9CNytQWgRya570=";
+  };
+
```

```suggestion

  dependencies = [ six ];
```

https://github.com/tehmaze/natural/blob/7d62ab06835096ca380b89e6204646a89a0b49da/setup.py#L47